src/current/cockroachcloud/private-clusters.md

```diff
@@ -24,15 +24,15 @@ During [limited access](/docs/{{site.versions["stable"]}}/cockroachdb-feature-av
 
 ## Create a private cluster
 
-To create a private cluster, you must use [{{ site.data.products.db }} API](cloud-api.html) or [CockroachDB's Terraform provider](https://registry.terraform.io/providers/cockroachdb/cockroach/latest/docs/resources/cluster). When you [create](https://cockroachlabs.com/docs/api/cloud/v1/clusters) a private cluster using Terraform Provider, you set its `private_network_visibility` field to `true`. This in turn automatically sets uses the Cloud API to set the cluster's `network_visibility` field to `NETWORK_VISIBILITY_PRIVATE`.
+On GCP, new {{ site.data.products.dedicated }} advanced clusters are private by default.
```

Comment:
On GCP, all dedicated clusters are private by default. Yeah it's confusing, but we've been in this soup since private clusters launched.

Reply:
Removed the word advanced from here